browser/brave_shields/ad_block_service_browsertest.cc

```diff
@@ -92,6 +92,11 @@ AdBlockServiceTest::~AdBlockServiceTest() {}
 void AdBlockServiceTest::SetUpOnMainThread() {
   ExtensionBrowserTest::SetUpOnMainThread();
   host_resolver()->AddRule("*", "127.0.0.1");
+  // Most tests are written for aggressive mode. Individual tests should reset
```

Comment:
It's not a good idea to override the default behavior like this, especially since this is commonly subclassed and this behavior will not be intuitive. Maybe you should split this into `AdblockServiceTest` and `AdblockServiceAggressiveModeTest` ?

Reply:
The reason I do it this way is because aggressive mode more closely matches the behavior of uBlock Origin, AdGuard, etc. which is what our upstream lists are primarily targeting; standard mode has some specific modifications to that behavior but it ultimately makes things _more_ complicated.

> Maybe you should split this into `AdblockServiceTest` and `AdblockServiceAggressiveModeTest`?

That does sound reasonable, although I still believe most subclasses should be hooking from the aggressive version.